capture-sdk/sdk/src/main/java/net/gini/android/capture/review/multipage/MultiPageReviewFragment.java

```diff
@@ -137,12 +139,9 @@ public void onCreate(final Bundle savedInstanceState) {
         OnBackPressedCallback callback = new OnBackPressedCallback(true /* enabled by default */) {
             @Override
             public void handleOnBackPressed() {
-                trackReviewScreenEvent(ReviewScreenEvent.BACK);
+                onBack();
```

Comment:
[Request] Here as well we should only track the back event:
```suggestion
                trackReviewScreenEvent(ReviewScreenEvent.BACK);
                setEnabled(false);
                remove();
```

Reply:
Here as well, same comment as above.

Reply:
Oh, I see! Then no changes needed. 👍